torch/ao/quantization/fx/_model_report/model_report_observer.py

```diff
@@ -30,9 +32,33 @@ def __init__(self):
         self.epoch_activation_min = torch.tensor(float("inf"))
         self.epoch_activation_max = torch.tensor(float("-inf"))
 
+        # keep track of per channel min max information using the given channel
+        self.ch_axis: int = ch_axis
+        self.min_val: torch.Tensor = torch.tensor([])
+        self.max_val: torch.Tensor = torch.tensor([])
+
     def forward(self, x):
         x_copy = x.detach()  # avoid keeping autograd tape
         x_copy = x_copy.to(self.epoch_activation_min.dtype)
+
+        # calculate the range stats
+        x_copy = self._calculate_range_stats(x_copy)
+
+        # calculate the min, max stats
+        x_copy = self._calculate_min_max_stats(x_copy)
```

Comment:
nit: These comments aren't helpful since they just repeat the method names. Best to remove them :)